public/components/search-field/date-search.pug

```diff
@@ -1,6 +1,5 @@
 .form-group
-    label {{title}}
-    
+ 
```

Comment:
Why did you remove the "Travel date" label?

Reply:
My mistake; it appears to be a redundant empty value on the Search page (deleting it removes the awkward space) but it wipes it out on the Explore page. I've restored it now.